samples/dashboards/demo/aws-management/demo.js

```diff
@@ -0,0 +1,949 @@
+let board = void 0;
+const pollingCheckbox = document.getElementById('enablePolling');
+
+pollingCheckbox.onchange = () => {
```

Comment:
Let's figure out the data for this demo to implement and test it properly.

Reply:
When the general concept of the demo is accepted, I will speak with Goran and upload the data on our public server.